pkgs/development/libraries/libspng/default.nix

```diff
@@ -32,13 +32,15 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" ];
 
+  strictDeps = true;
+
   nativeCheckInputs = [
     cmake
-    libpng
   ];
 
   buildInputs = [
     zlib
+    libpng
```

Comment:
spng advertises itself as an alternative, so why would it be a dependency?

Reply:
And you need to add the error message to the  commit message

Reply:
Updated to be more descriptive

Reply:
Wouldn't it be checkInputs?

Reply:
Moving `libpng` to `checkInputs` results in the same error